addons/mass_mailing_sms/tests/test_mailing_sms_ui.py

```diff
@@ -0,0 +1,43 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import odoo.tests
+
+
+@odoo.tests.tagged('post_install', '-at_install')
+class TestUi(odoo.tests.HttpCase):
+
+    def test_mailing_mailing_activities_split_in_systray(self):
```

Comment:
Use the same name as the tour, like `test_tour_mailing_activities_split`, that way when we grep the tour name we get everything related to it in the code.